server/service/src/demographic/update_demographic_projection.rs

```diff
@@ -55,22 +55,15 @@ pub fn validate(
     connection: &StorageConnection,
     input: &UpdateDemographicProjection,
 ) -> Result<DemographicProjectionRow, UpdateDemographicProjectionError> {
-    let demographioc_projection_row =
+    let demographic_projection_row =
         match check_demographic_projection_exists(&input.id, connection)? {
             Some(demographic_projection_row) => demographic_projection_row,
             None => {
                 return Err(UpdateDemographicProjectionError::DemographicProjectionDoesNotExist)
             }
         };
-    if let Some(base_year) = input.base_year {
-        if !check_base_year_unique(base_year, connection)? {
```

Comment:
This is still breaking the tests... I think should put this check back - especially now that  the `getByBaseYear` call is returning only one projection!

Reply:
The reason this was removed is because it prevents updates from happening!
it checks to see if `baseYear` is anywhere in the table - and of course it will be if you are updating a row and only changing the percentages. 

What it should check, is whether any rows other than this one have this base year. I'll update the validation and tests.